adcc/adc_pp/state2state_transition_dm.py

```diff
@@ -31,7 +31,7 @@
 
 
 def s2s_tdm_adc0(mp, amplitude_l, amplitude_r, intermediates):
-    check_singles_amplitudes(amplitude_l, amplitude_r)
+    check_singles_amplitudes(amplitude_l, amplitude_r, spaces=[b.o, b.v])
```

Comment:
Sorry to be a bother ... but for this one I would prefer brevity, so
```suggestion
    check_singles_amplitudes(amplitude_l, amplitude_r, [b.o, b.v])
```

Reply:
Won't work this way because of the `*amplitudes` argument...

Reply:
ah ... yikes you are right. That's annoying. Ok than leave it as is.

Reply:
I committed another suggestion 😄